gradle/libs.versions.toml

```
@@ -0,0 +1,27 @@
+[versions]
+# this is our own version we deploy
+baseVersion =  "1.29.2"
+
+#versions
+kotlinApi = "1.7"
+kotlin = "1.7.10"
+kotlinJVMtarget = "1.8"
+
+# plugins
+kotlinCompatibilityValidator = "0.13.2"
+
+# libraries
+semver = "1.1.2"
+junit = "4.13.2"
+
+git-based-versioning = "*"
+buildBackends = "2.4.0+"
```

Some names in this file use dashes, some use camel case, this should be made consistent.

---

Also, the correct syntax is 2.4.0.+, Gradle is confusing here.
```suggestion
buildBackends = "2.4.0.+"
```

---

Good catch! I accidentally dropped the dot in my change.